pytorch_lightning/logging/test_tube_logger.py

```diff
@@ -77,6 +77,13 @@ def rank(self, value):
         if self._experiment is not None:
             self.experiment.rank = value
 
+    @property
+    def name(self):
+        if self._experiment is None:
```

Comment:
rather:
```
name = self._name if self._experiment is None else self.experiment.name
return name
```

Reply:
IMHO the current version is easier to read.